es6-async-await/exceptions.js

```diff
@@ -0,0 +1,56 @@
+import fetch from './fetch.js';
+
+// Keep track of app load time so each log message can be timed.
+// Log messages should all be approximately 1 second apart.
+const startTime = Date.now();
+const elapsed = () => `${Math.round((Date.now() - startTime) / 1000)}s -`;
+
+function throwOnce() {
```

Comment:
This file wasn't converted. It should use async/await with try/catch blocks.

Also, there's no exceptions-best.js.

Reply:
My fault I got trigger-happy and didn't finish the full assignment! My apologies!